auto_nag/scripts/tracked_bad_severity.py

```diff
@@ -59,7 +55,12 @@ def get_bz_params(self, date):
         return params
 
     def get_autofix_change(self):
-        return {"severity": "major"}
+        return {
+            "comment": {
+                "body": f"This bug is tracked but with a small severity so change it to major.\n{self.get_documentation()}"
```

Comment:
```suggestion
                "body": f"This bug is tracked by a release manager but with a small severity so change it to major.\n{self.get_documentation()}"
```

Reply:
For this tool we don't check if a release manager changed the flag (even if it is very likely).

Reply:
the tracking action is done only by release manager

Reply:
@calixteman ping?